cmake/FindLAPACKE.cmake

```diff
@@ -0,0 +1,380 @@
+#                          of Tennessee Research Foundation.
+#                          All rights reserved.
+# @copyright (c) 2012-2016 Inria. All rights reserved.
+# @copyright (c) 2012-2014 Bordeaux INP, CNRS (LaBRI UMR 5800), Inria, Univ. Bordeaux. All rights reserved.
+#
+###
+#
+# - Find LAPACKE include dirs and libraries
+# Use this module by invoking find_package with the form:
+#  find_package(LAPACKE
+#               [REQUIRED] # Fail with error if lapacke is not found
+#               [COMPONENTS <comp1> <comp2> ...] # dependencies
+#              )
+#
+#  LAPACKE depends on the following libraries:
+#   - LAPACK
+#
+# This module finds headers and lapacke library.
+# Results are reported in variables:
+#  LAPACKE_FOUND            - True if headers and requested libraries were found
+#  LAPACKE_LINKER_FLAGS     - list of required linker flags (excluding -l and -L)
+#  LAPACKE_INCLUDE_DIRS     - lapacke include directories
+#  LAPACKE_LIBRARY_DIRS     - Link directories for lapacke libraries
+#  LAPACKE_LIBRARIES        - lapacke component libraries to be linked
+#  LAPACKE_INCLUDE_DIRS_DEP - lapacke + dependencies include directories
+#  LAPACKE_LIBRARY_DIRS_DEP - lapacke + dependencies link directories
+#  LAPACKE_LIBRARIES_DEP    - lapacke libraries + dependencies
+#
+# The user can give specific paths where to find the libraries adding cmake
+# options at configure (ex: cmake path/to/project -DLAPACKE_DIR=path/to/lapacke):
+#  LAPACKE_DIR             - Where to find the base directory of lapacke
+#  LAPACKE_INCDIR          - Where to find the header files
+#  LAPACKE_LIBDIR          - Where to find the library files
+# The module can also look for the following environment variables if paths
+# are not given as cmake variable: LAPACKE_DIR, LAPACKE_INCDIR, LAPACKE_LIBDIR
+#
+# LAPACKE could be directly embedded in LAPACK library (ex: Intel MKL) so that
+# we test a lapacke function with the lapack libraries found and set LAPACKE
+# variables to LAPACK ones if test is successful. To skip this feature and
+# look for a stand alone lapacke, please add the following in your
+# CMakeLists.txt before to call find_package(LAPACKE):
+# set(LAPACKE_STANDALONE TRUE)
+
+#=============================================================================
+# Copyright 2012-2013 Inria
+# Copyright 2012-2013 Emmanuel Agullo
+# Copyright 2012-2013 Mathieu Faverge
+# Copyright 2012      Cedric Castagnede
+# Copyright 2013-2016 Florent Pruvost
+#
+# Distributed under the OSI-approved BSD License (the "License");
+# see accompanying file MORSE-Copyright.txt for details.
+#
+# This software is distributed WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the License for more information.
+#=============================================================================
+# (To distribute this file outside of Morse, substitute the full
+#  License text for the above reference.)
```

Comment:
We need to include the full license as request here.  Also can we add to the `LICENSE` file, see for example the `FindHCC.cmake` section.

Reply:
Oh. Good point. I'll try and fix that one now.

Reply:
Done.